### PR TITLE
Update saml2auth and datagovtheme

### PR DESCRIPTION
**Pull request by thejuliekramer**
Updates catalog with changes made in ckanext-saml2auth https://github.com/gsa/datagov-deploy/issues/2526 and ckanext-datagovtheme https://github.com/GSA/datagov-deploy/issues/2465

### CODE REVIEW

File: ckan/requirements.txt [hunk 1]
```diff
@@ -13,7 +13,7 @@ chardet==3.0.4
 -e git+https://github.com/ckan/ckan.git@eca78d5df65414249c5a76620c1acae1ddc76cc3#egg=ckan
 -e git+https://github.com/ckan/ckanext-archiver.git@4cb10ac9f96fc78bc7be4b3dee8fbc56049b2865#egg=ckanext-archiver
 -e git+https://github.com/GSA/ckanext-datagovcatalog.git@531d20c925b4b8954e71a25f992ebbf9000b4c30#egg=ckanext-datagovcatalog
--e git+https://github.com/GSA/ckanext-datagovtheme.git@a89917e07f7df106399192c79166bf92aceb0e71#egg=ckanext-datagovtheme
+-e git+https://github.com/GSA/ckanext-datagovtheme.git@32b890a7c24dfbe58f01e46f5003325d70b2fb45#egg=ckanext-datagovtheme
```
Comment (FuhuXia) [inline ckan/requirements.txt:16]:
```suggestion
-e git+https://github.com/GSA/ckanext-datagovtheme.git@master#egg=ckanext-datagovtheme
```

File: ckan/requirements.txt [hunk 2]
```diff
@@ -22,7 +22,7 @@ ckanext-envvars==0.0.1
 -e git+https://github.com/ckan/ckanext-harvest.git@d00553831e20ae7c6737206ddb8a613b494f2b86#egg=ckanext-harvest
 -e git+https://github.com/ckan/ckanext-qa.git@d7d384cb18a85243ea62ef0bcc76bc1775f1c806#egg=ckanext-qa
 -e git+https://github.com/davidread/ckanext-report.git@b67875b2a5b4c9b9ab2cf255f074226255ca9398#egg=ckanext-report
--e git+https://github.com/keitaroinc/ckanext-saml2auth.git@d9d7db603fe41b2c4c35ab21fdaf8ea005ad5f7c#egg=ckanext-saml2auth
+-e git+https://github.com/keitaroinc/ckanext-saml2auth.git@a9bc575be31d2099650e6ca9a3b484bc0034717e#egg=ckanext-saml2auth
```
Comment (FuhuXia) [inline ckan/requirements.txt:25]:
```suggestion
-e git+https://github.com/keitaroinc/ckanext-saml2auth.git@ckan-2.8#egg=ckanext-saml2auth
```

Comment (avdata99) [reply ckan/requirements.txt:25]: @FuhuXia the requirements.txt file is self-generated. We don't suppose to change it. We just change the `toml` file

File: ckan/requirements.txt (hunk 1 above)
Comment (avdata99) [reply ckan/requirements.txt:16]: @FuhuXia the requirements.txt file is self-generated. We don't suppose to change it. We just change the `toml` file

**Comment by FuhuXia** [comment]
Can we use branch names, so we don't need to come back and update commit hashs?

**Comment by avdata99** [comment]
> Can we use branch names, so we don't need to come back and update commit hashs?

No @FuhuXia, if we do that new version of all libraries will be deployed without testing

**Comment by FuhuXia** [comment]
I see. I thought it was same structure of requirements.txt and  requirements-freeze.txt. Obviously not.

**Pull request merged**